### ISSUE TITLE
ARE YOU GOING TO FIX THE MESSENGER OR NOT

### ISSUE THREAD

**Issue by 2557516** [bdsmlr-development/bdsmlr-updates#8]
Your refusal to address the fact that the website's chat function DOES NOT WORK AT ALL for months now despite every one of your issue boards and tweets being swamped in people asking you why it's broken is irritating. 

Are you going to fix it or not? If not, just delete the feature off the site so we know you have no intention of it being an interactive platform and we can be done.

**Comment by tiyb** [comment]
I'd echo that, but please note: This is the existing messenger, not the weird tag-based thing currently being worked on. The ability to message another user on the site, directly.

**Comment by Mack6767** [comment]
Myself as well as a number of other blogs I've spoken with are leaving your site until the issue is resolved. It's a great place to be if it were 100% working, but since a crucial part is failing and there seems to be no addressing of the issue, we feel we have no other recourse than to find an alternative site.

**Comment by stereopandaa** [comment]
agreed, until this issue isn't solved properly, leaving the site is the only alternative. bdsmlr is good, but if that is not working as it should, there is only leaving the site, and lots of people are doing that.

**Comment by Scott16546** [comment]
It's still broken I'm sending chats but no one gets my messages
